pkg/eks/fargate.go

```diff
@@ -3,29 +3,35 @@ package eks
 import (
 	"time"
 
+	"github.com/aws/aws-sdk-go/service/eks"
 	"github.com/kris-nova/logger"
 	"github.com/pkg/errors"
 	"k8s.io/client-go/kubernetes"
 
-	"github.com/weaveworks/eksctl/pkg/fargate"
 	"github.com/weaveworks/eksctl/pkg/fargate/coredns"
 	"github.com/weaveworks/eksctl/pkg/utils/retry"
 	"github.com/weaveworks/eksctl/pkg/utils/strings"
 
 	api "github.com/weaveworks/eksctl/pkg/apis/eksctl.io/v1alpha5"
 )
 
+//go:generate counterfeiter -o fakes/fargate_client.go . FargateClient
```

Comment:
This is already broken in `master` from https://github.com/weaveworks/eksctl/pull/2766/files, I missed it :cry:
`counterfeiter` has to be in `tools.go` and this should be:

```suggestion
//go:generate "${GOBIN}/counterfeiter" -o fakes/fargate_client.go . FargateClient
```

Reply:
Gah! Nice spot, I will also throw in a fix for the other one for free.

Reply:
Hmmm does not seem to like that :(, do you know if I need to set anything else up to make it work in CI?

Reply:
Ah nice one, forgot about `.requirements`!